packages/manager/core/generator/app/templates/vitest.config.js

```diff
@@ -0,0 +1,34 @@
+import path from 'path';
+import { defineConfig } from 'vite';
+import react from '@vitejs/plugin-react';
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [react()],
+  test: {
+    globals: true,
+    environment: 'jsdom',
+    setupFiles: [],
+    coverage: {
+      include: ['src'],
+      exclude: [],
+    },
+    testTimeout: 60000,
```

Comment:
```suggestion
    testTimeout: 60_000,
```

May be more readable for all future project

Reply:
👍